contracts/base/strategies/iron-fold/IronFoldStrategyBase.sol

```diff
@@ -141,6 +148,7 @@ abstract contract IronFoldStrategyBase is StrategyBase {
   function doHardWork() external onlyNotPausedInvesting override restricted {
     claimReward();
     liquidateReward();
+    investAllUnderlying();
```

Comment:
btw better to invest before liquidation coz probably other strategies can have the same underlying as reward token

Reply:
If you have underlying=reward and invest first, no tokens will ever be used for buyback.